houses/views.py

```diff
@@ -1,3 +1,70 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db        import transaction
+
+from houses.models    import Country, Ghost, House, HouseImage, HouseType, City
+from core.s3_storages import s3_client
+from users.utils      import token_auth
+
+class BeingTheHostView(View):
```

Comment:
class명으 `Host` 만으로 충분합니다.
```suggestion
class HostView(View):
```

Reply:
반영해서 수정했습니다~!